tip-calculator/src/App.tsx

```diff
@@ -1,9 +1,41 @@
-import React from 'react'
+import React from 'react';
+
+import Input from './components/Input';
+
+import dollar from './assets/icon-dollar.svg';
+import person from './assets/icon-person.svg';
 
 const App = () => {
+  const handleChange = (value: string | number) => {
+    console.log(value);
+  };
+
   return (
-    <div>App</div>
-  )
-}
+    <div>
+      App
+      <Input
+        label="Bill"
+        type="float"
+        value={230}
+        icon={dollar}
+        onChange={handleChange}
+      />
+      <Input
+        label="Number of People"
+        type="integar"
```

Comment:
Type should be integer

Reply:
Typo has been updated. I was also required to update jest and add a mock for my svg files as this repo now has a pre-commit to test files